### ISSUE TITLE
navigation button submits form

### ISSUE THREAD

**Issue by dranitski** [josdejong/svelte-jsoneditor#39]
hey, your navigation button have this html:
```
<button class="jse-navigation-bar-button jse-navigation-bar-arrow svelte-lfr3gu">...</button>
```
so, it dont have `type = button` attr specified. That means browsers threat such buttons as `submit` buttons (proof https://stackoverflow.com/a/9643866/10099510)

I use your json-editor as a part of bootstrap form so it makes some trouble in our case :grinning:

**Comment by dranitski** [comment]
https://github.com/josdejong/svelte-jsoneditor/blob/main/src/lib/components/controls/navigationBar/NavigationBarDropdown.svelte#L16-L30

**Comment by josdejong** [comment]
Thanks, it is so easy to forget. I'm looking around for an eslint rule to guard against forgetting `type` on a `<button>` but didn't yet find it (I asked here: https://github.com/sveltejs/eslint-plugin-svelte3/issues/151).

I just went through the code and added `type` where missing

**Comment by ota-meshi** [comment]
Hi @josdejong.
I'm trying an experimental plugin that can report it. However, this plugin cannot be used with the official eslint-plugin-svelte3. Also, it's still unclear if the experimental plugin is better than the official plugin.
It would be great if you could experiment with me to see if it works.

https://ota-meshi.github.io/eslint-plugin-svelte/rules/button-has-type/

I'm sorry if this comment feels like spam.

**Comment by josdejong** [comment]
ah, nice job @ota-meshi ! 

At this point I would like to stick with the "official" plugin (using svelte-kit beta is enough living on the edge for me 😉 ). It would be great if your work could be merged or unified somehow with `eslint-plugin-svelte3`, instead of putting effort in two different initiatives forcing the user to chose.

**Comment by ota-meshi** [comment]
Thank you for your opinion. It's makes sense to use official plugin :+1:.   
These plugins are difficult to merge due to their different mechanics, but I hope I can work with them. I would like to investigate Pros / Cons for that.

**Comment by josdejong** [comment]
😎

**Comment by josdejong** [comment]
Fix is published now in `v0.3.10` (we don't yet have a linter rule in place though)
